docs/src/examples/solvers/Unicycle robot copy.jl

```diff
@@ -0,0 +1,251 @@
+using Test     #src
```

Comment:
What is this file for ?

Reply:
This is the correct file for now. Actually, I've done the first version where I repeat the x and u variables according to N (the horizon) `@variable(model, x_low[i] <= x[i = 1:3, 1:N] <= x_upp[i], start = x_initial[i])`, but there's a bug with N (but I think we just need something like set_attribute N). I raised this [bug](https://github.com/dionysos-dev/Dionysos.jl/issues/384). I will finish the first simple version of unicycle robot without the horizon that is only needed for stage cost and terminal cost.